Source/Solocast/Solocast.DAL/SQLite/SQLitePodcastService.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Newtonsoft.Json;
+using Solocast.Core.Contracts;
+using Solocast.Core.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Storage;
+using Solocast.DAL.SQLite;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Microsoft.EntityFrameworkCore;
+
+namespace Solocast.DAL
+{
+    public class SQLitePodcastService : ILocalStorageService<Podcast>
+    {
+        public void Migrate()
+        {
+            using (var db = new PodcastsContext())
+            {
+                db.Database.Migrate();
+            }
+        }
+
+        public async Task<IList<Podcast>> LoadAsync()
+        {
+            List<Podcast> entities;
+
+            using (var db = new PodcastsContext())
+            {
+                entities = db.Podcasts.Include(x=>x.Episodes).ToList();
```

Comment:
I think EF has a `ToListAsync()` extension method. You can try using that to avoid calling `await Task.FromResult()`